test/Mono.Linker.Tests.Cases/LinkXml/CanPreserveMarkedChildrenOfUnrequiredType.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+using Mono.Linker.Tests.Cases.LinkXml.Dependencies;
+
+namespace Mono.Linker.Tests.Cases.LinkXml
+{
+	[SetupCompileBefore ("Library.dll", new [] { "Dependencies/CanPreserveMarkedChildrenOfUnrequiredType_Library.cs" })]
```

Comment:
I don't see a reason why this test needs to setup an additional assembly.  I believe it could all be in the test assembly.

Reply:
You're right. Thanks. Please take a look: #1019.